lib/main_common.pm

```diff
@@ -255,6 +256,17 @@ sub load_slepos_tests {
     }
 }
 
+sub load_docker_tests {
+    loadtest "console/docker";
+    loadtest "console/runc";
+    # No package 'docker-compose' in SLE
+    if (check_var('DISTRI', 'opensuse')) {
+        loadtest "console/docker_compose";
+    }
+    if (check_var('DISTRI', 'sle') && !sle_version_at_least('15')) {
```

Comment:
I think that `is_sle` function could be called instead of `check_var(...)`, no? ;-)